www/app/main/main.controller.js

```diff
@@ -3,8 +3,11 @@
     .module('app')
     .controller('MainController', MainController);
 
-  function MainController() {
+  MainController.$inject = ['usersService'];
+
+  function MainController(usersService) {
     var vm = this;
     vm.name = '';
```

Comment:
Our db uses the variable username for the user. Shall we update the DB or this file to remain consistent?